dbms/src/Common/RadixSort.h

```diff
@@ -163,6 +180,8 @@ struct RadixSort
     using CountType   = typename Traits::CountType;
     using KeyBits     = typename Traits::KeyBits;
 
+    static constexpr size_t INSERT_SORT_THRESHOLD = 64;
```

Comment:
Missing comment (what it is; the motivation for constant: how it was derived)
PS. It is names "insertion sort", not "insert sort".

Reply:
Added a comment.
This constant is used in kxsort. I did not attempt to optimize it.